src/components/events/FederatedRSVPButton.tsx

```
@@ -0,0 +1,80 @@
+    setError(null);
+
+    try {
+      const { data, error: funcError } = await supabase.functions.invoke(
+        "federated-oauth-handshake",
+        {
+          body: {
+            origin_domain: originDomain,
+            origin_event_id: originEventId,
+            mentee_or_attendee_email: userEmail,
+          },
+        }
```

ESLint

[error] Insert `,`

(prettier/prettier)